### ISSUE TITLE
Pydantic Version Error

### ISSUE THREAD

**Issue by moluwole** [graphql-python/graphene-pydantic#43]
When installing graphene-pydantic with pip, I get this error message
```
ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

graphene-pydantic 0.1.0 requires pydantic<=1.6,>=1.0, but you'll have pydantic 1.6.1 which is incompatible.
```

This does not affect installation in any way but it would help updating the library to use the latest pydantic version

**Comment by kierandarcy** [comment]
Pydantic is no v1.61. Could the Poetry dependency be pinned to >=1.0 only (without the upper boundary)?

```
[tool.poetry.dependencies]
pydantic = ">=1.0"
```

**Comment by necaris** [comment]
@kierandarcy are you able to confirm things work as expected with `1.6.1`? We have generally been wary of pinning to any future version of a library because `graphene-pydantic` messes around with internal APIs of `pydantic` that aren't expected to remain backward compatible.

**Comment by kierandarcy** [comment]
Hi @necaris I have not yet run the `graphene-pydantic` test suite, but my application is running without issues with `pydantic 1.6.1`. I have a little this afternoon, I'll get a `graphene-pydantic` dev environment set up, run the test suite and let you know the outcome.

**Comment by kierandarcy** [comment]
>  I'll get a `graphene-pydantic` dev environment set up, run the test suite and let you know the outcome.

All tests pass.

It would be awesome if you could pin Pydantic to >=1.0,<2.0.

**Comment by necaris** [comment]
@kierandarcy since Pydantic has had breaking changes in minor versions before, and `graphene-pydantic` uses internal APIs that have no compatibility guarantees, I would be more comfortable pinning to `>=1.0,<1.7` until we can test against a future `1.7` release. 

Any chance you'd be interested in submitting this change as a pull request? It might even count towards [Hacktoberfest](https://hacktoberfest.digitalocean.com/) if that's interesting to you :smile:

**Comment by kierandarcy** [comment]
`Any chance you'd be interested in submitting this change as a pull request?`

No problem at all, and delighted to do so.

**Comment by necaris** [comment]
Closed by #44, thanks @kierandarcy !
